gnmawsgr/tasks.py

```diff
@@ -129,11 +129,19 @@ def valid_fields(self):
 def download_callback(rq, current_progress,total):
     if rq.status != 'DOWNLOADING':
         rq.status = 'DOWNLOADING'
-    logger.info("{itemid} Download in progress: {cur}/{tot}, {pc:.2f}%".format(
-        itemid=rq.item_id,
-        cur=current_progress,
-        tot=total,
-        pc=100*float(current_progress)/float(total)))
+
+    try:
+        percent = 100*float(current_progress)/float(total)
+
+    except ZeroDivisionError as e:
+        percent = 0.0
+        logger.warning("{0} Trying to download but last chuck size was zero".format(rq.item_id))
+
+    #logger.info("{itemid} Download in progress: {cur}/{tot}, {pc:.2f}%".format(
```

Comment:
best not to comment this out, just return from the exception handler above and leave this in